src/main/java/org/jenkinsci/plugins/environmentdashboard/DashboardBuilder.java

```diff
@@ -172,7 +180,7 @@ public void tearDown(Run<?, ?> build, FilePath workspace, Launcher launcher, Tas
 		context.setDisposer(new TearDownImpl());
 	}
 
-	@SuppressWarnings("rawtypes")
```

Comment:
why did you remove this suppress?

Reply:
I've added it back. I removed it during code modifications.